docs/features/startup-jobs.md

```diff
@@ -0,0 +1,72 @@
+# Running Startup Jobs
+
+**NCronJob** allows you to configure jobs to run at application startup. This is useful for tasks that need to be executed immediately when the application starts, such as initial data loading, cleanup tasks, or other setup procedures.
+
+Startup jobs are defined like regular CRON jobs and inherit from `IJob`. The only difference is that they are configured to run at startup.
```

Comment:
Probably we can also mention that **all** startup jobs have to be completed before any other CRON (or instant) job starts.